vector/build.gradle

```diff
@@ -465,8 +451,6 @@ dependencies {
         exclude group: "org.jetbrains.kotlinx", module: "kotlinx-coroutines-debug"
     }
 
-<<<<<<< HEAD
-
```

Comment:
All the following lines should probably be moved to `vector-app/build.gradle` if this is the case in Element